<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>627</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RSA Encryption</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>261</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AES Text Encryption</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>781</width>
      <height>511</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Encrypt</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Your Text :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="inputText_txt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>761</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="encrypt_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>621</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypt</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="outputText_txt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>210</y>
        <width>711</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Encrypted Text :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>270</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Your IV :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="iv_output_txt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>711</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="clearencrypt_btn">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>140</y>
        <width>131</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear All</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>49</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Key :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="encrypt_key_text">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>571</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="generateky_btn">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>20</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
     <widget class="QPushButton" name="copyKey_btn">
      <property name="geometry">
       <rect>
        <x>700</x>
        <y>20</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Copy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="copyEncrypted_btn">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>210</y>
        <width>41</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Copy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="copyIv_btn">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>290</y>
        <width>41</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Copy</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>761</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QTextEdit" name="log_encrypt_txt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>370</y>
        <width>691</width>
        <height>101</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Log :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_encrypt_log_btn">
      <property name="geometry">
       <rect>
        <x>710</x>
        <y>370</y>
        <width>61</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Log</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Decrypt</string>
     </attribute>
     <widget class="QTextEdit" name="inputDecKey_txt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>761</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Your Key :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="inputDecIv_txt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>761</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Your IV :</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="inputDecText_txt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>761</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Your Text :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="decrypt_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>761</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Dencrypt</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="outputDecrypt_txt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>260</y>
        <width>671</width>
        <height>211</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="copyDecrypted_btn">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>260</y>
        <width>81</width>
        <height>211</height>
       </rect>
      </property>
      <property name="text">
       <string>Copy</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
